src/Domain/MetaCampaignRepository.php

```diff
@@ -24,4 +24,25 @@ public function getBySlug(MetaCampaignSlug $slug): ?MetaCampaign
     {
         return $this->doctrineRepository->findOneBy(['slug' => $slug->slug]);
     }
+
+    public function countCompleteDonationsToMetaCampaign(MetaCampaign $metaCampaign): int
```

Comment:
I think counting is probably quick enough that it's OK, but we could consider speeding this up by denormalising the slug (or in future metacampaign id) onto the donation record so that we don't need to join, and/or putting the result behind the symfony cache.

Neither done so far.